### PR TITLE
Implement the placeholder to the platform and the whitepaper

### PR DESCRIPTION
**Pull request by MostafaBalata**
Create [whitepaper, product, FAQ] pages and make the footer stick to the bottom always

### CODE REVIEW

File: page/index.ejs [hunk 1]
```diff
@@ -1,74 +1,14 @@
...
-            
-            <%- checkSectionVisiablity(content.ecosystem,
-            include('partials/ecosystem',  content.ecosystem )); %>
-
+        <div class="main container">
+                        
+            <%- include('partials/home' ); %>
             <%- include('partials/footer', content.footer); %>
             <script src="/main.js"></script>
+
```
Comment (Moejoe90) [inline page/index.ejs]: Extra black spaces

File: page/sass/footer.scss [hunk 2]
```diff
@@ -13,13 +21,17 @@
...
 }
-
+.border-top{
+    border-top: 1px $off-white solid;
+    padding-top: 20px;
+}
 .footer{
-    padding-top: $padding-between-sections;
+    padding-top: $padding-between-sections;   
+    border:0 !important;
```
Comment (Moejoe90) [inline page/sass/footer.scss]: Why is important necessary, is there something overriding this even after stating style normally?

File: page/sass/whitepaper.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.whitepaper{
+    border:0px;
```
Comment (Moejoe90) [inline page/sass/whitepaper.scss]: Should be a space between

File: page/partials/header.ejs [hunk 4]
```diff
@@ -7,18 +7,18 @@
         <span class="icon-bar"></span>
```
Comment (Moejoe90) [inline page/partials/header.ejs]: Active page should have black color, 
currently it appears as if commit is always active

File: app/index.tsx [hunk 5]
```diff
@@ -11,12 +11,14 @@ import Countdown from "./containers/Countdown";
 const root = document.getElementById("react-root");
 
 const render = () => {
-  ReactDOM.render(
+  if (root) {
```
Comment (krzkaczor) [inline app/index.tsx]: could you leave additional comment here that we're doing it because of the fact that we load whole bundle on each subpage?

**Pull request merged**